src/home/HomeButtons.js

```diff
@@ -41,4 +41,113 @@ export const FlexItem = styled(Button)`
     width: 200px;
     height: auto;
   }
-`;
\ No newline at end of file
+`;
+
+export const ButtonAchievements = styled.button`
+  border-radius: 4px;
+  background: #ffc107;
+  margin: 20px;
+  padding: 20px;
+  border-radius: 10px;
+  width: 400px;
+  height: 100px;
+  border: none;
+  color: #000000;
+  text-align: center;
+  font-size: 20px;
+  transition: all 0.5s;
+  cursor: pointer;
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  transition: 0.5s;
+  :after {
+  content: '🏆';
+  position: absolute;
+  opacity: 0;  
+  top: 36px;
+  right: -20px;
+  transition: 0.5s;
+  }
+  :hover{
+  padding-right: 25px;
+  padding-left:8px;
+  }
+  :hover:after {
+  opacity: 1;
+  right: 115px;
+  }
+`
+
+export const ButtonStart = styled(Button)`
+  border-radius: 4px;
+  background-color: #28a745;
+  margin: 20px;
+  padding: 20px;
+  border-radius: 10px;
+  width: 400px;
+  height: 100px;
+  border: none;
+  color: #fff;
+  text-align: center;
+  font-size: 20px;
+  transition: all 0.5s;
+  cursor: pointer;
+  cursor: pointer;
```

Comment:
duplicate

Reply:
fixed

Reply:
looks like all the buttons have this issue

Reply:
done